internal/x/net/dns_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
possibly missing a build tag?

```suggestion

//go:build integration

```

Reply:
@Al-Pragliola ping

Reply:
done in https://github.com/sighupio/furyctl/pull/198/commits/4e8517d3118c6a4854ff7f8bee4d883f8bcc17b6